web/src/pages/people/index.tsx

```diff
@@ -45,6 +55,30 @@ export default function PeoplePage() {
   const isIAO = userData?.roles.includes("information-asset-owner");
   const isTreOpsStaff = userData?.roles.includes("tre-ops-staff");
 
+  const handleUserSearch = async (query: string) => {
+    if (query !== "") {
+      const response = await getUsers({ query: { find: query } });
+      if (response.response.ok && response.data) {
+        setUsers(response.data);
+      } else {
+        setSearchErrorMessage("Oops, something went wrong with your search. Refresh and try again");
+      }
+    } else {
+      setUsers(originalUsers);
+    }
+  };
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchTerm(event.target.value);
+    if (event.target.value === "") {
+      setUsers(originalUsers);
+    }
+  };
```

Comment:
Its not essential (more of a nice to have) but it might be more efficient to attach the search term to a `useRef`. That way, the state isn't being updated with every keystroke and is just used as an when it is needed. 

I don't feel strongly about implementing it in this alternative way. But thought you might be interested to know about it.